### PR TITLE
Add a new function for multiple pkg version checking

### PR DESCRIPTION
**Pull request by xiaodwan**
Sometime a case may requires multiple pkgs are at special versions.
This new function can help you specify those pkgs and their versions
in cases cfg files.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.log_size_decrease.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1743.84 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @yantian0 Please help to  review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -1836,8 +1836,37 @@ def compare_version(interval, version=None, cmd=None):
     """
     if not version:
         if not cmd:
-            cmd = 'rpm -q libguestfs'
+            cmd = 'rpm -q virt-v2v'
```
Comment (yantian0) [inline virttest/utils_v2v.py:1839]: Is it required to distinguish the rpm based on the release version?

Comment (xiaodwan) [reply virttest/utils_v2v.py:1839]: No, This is a default behaviour change. 
Users should always tell the api what cmd they are using like the new function does.

**Review by chunfuwen — APPROVED** [review]

**Review by chloerh — APPROVED** [review]

File: virttest/utils_v2v.py (hunk 1 above)
Comment (yantian0) [reply virttest/utils_v2v.py:1839]: Ok, got it. Thank you.

**Review by yantian0 — APPROVED** [review]

**Pull request merged**